kahuna/public/js/image/service.js

```diff
@@ -25,11 +25,16 @@ imageService.factory('imageService', [function() {
     }
 
     function getStates(image) {
+        const persistReasons = image.data.persisted.reasons;
+
         return {
             cost: image.data.cost,
             hasCrops: hasExportsOfType(image, 'crop'),
             isValid: image.data.valid,
-            canDelete: image.getAction('delete').then(action => !! action)
+            canDelete: image.getAction('delete').then(action => !! action),
+            canArchive: image.data.persisted.value === false ||
```

Comment:
> So are we saying you can't archive this if it's already persisted?

Yep. Archiving an image that already has a crop has no real effect, other than possibly going the user reassurance that it won't be deleted. If we remove/lock the option of archiving in this scenario, then it means less clicks for the user.

> That's not true on the API level is it?

Correct. The API allows you to POST to archived regardless of the value of persisted.